[docs] Refactor "Selectors" (#4799)
DevExpress/testcafe: docs/articles/documentation/reference/test-api/selector/parent.md

---
layout: docs
title: Selector.parent Method
permalink: /documentation/reference/test-api/selector/parent.html
---
# Selector.parent Method

Selects parent ***elements***.

## Syntax

### parent

```text
Selector().parent() → Selector
```

> Finds the parents of all nodes in the matched set (first element in the set will be the closest parent).

Reviewer: ```suggestion
Finds the parents of all nodes in the matched set (first element in the set is the closest parent).
```
